.github/ISSUE_TEMPLATE/blank-issue.md

```diff
@@ -1,19 +1,23 @@
 ---
 name: Blank Issue
-about: Consistent formatting make Issues concise and easy to navigate
+about: Consistent formatting makes Issues concise and easy to navigate.
 title: ''
 labels: ''
 assignees: ''
 
 ---
 
 ### Overview
-REPLACE THIS TEXT -Text here that clearly states the purpose of this issue in 2 sentences or less.
+Briefly describe the purpose of this issue. 
```

Comment:
hmm... the REPLACE... at the start of each line might be useful later on.

It allows a github action to auto reject an issue for not completing all the parts. these templates are copied from a set that hack for LA maintains and updates. I think it's fine to change their wording, just maybe have some common words to identify what still needs filling out.

Reply:
Thanks for providing the context around this. 

> REPLACE THIS TEXT -Text here that clearly states the purpose of this issue in 2 sentences or less.

It looked tacky to me, but I can live with it if it serves a purpose. I looked at the `workflows` within `.github` but am only seeing one workflow, though (i.e., `new-issue-create-card.yml`)

In fact, I just created Test Issue #116 and I think I just reverse engineered creating an issue without following any of the issue templates... 😅